rapids_config.cmake

```diff
@@ -12,6 +12,8 @@
 # the License.
 # =============================================================================
 file(READ "${CMAKE_CURRENT_LIST_DIR}/VERSION" _rapids_version)
+set(rapids-cmake-repo "vyasr/rapids-cmake")
```

Comment:
Is that right?
```suggestion
set(rapids-cmake-repo "feat/rapids-cmake")
```

Reply:
This is correct. It's pointing at my fork of rapids-cmake. The line below is the branch. However, this will get reverted before we merge, it's only there right now for testing purposes.